src/main/java/bio/terra/janitor/service/cleanup/CleanupParams.java

```diff
@@ -0,0 +1,25 @@
+package bio.terra.janitor.service.cleanup;
+
+import bio.terra.generated.model.CloudResourceUid;
+import bio.terra.janitor.db.TrackedResourceId;
+import bio.terra.stairway.FlightMap;
+
+/** /** Helper class to manage input parameters in a {@link FlightMap}. */
```

Comment:
remove duplicate `/**` and add `in a {@link FlightMap} for all cleanup flights. */`